_releases/2023-07-11-1.9.0-released.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Crystal 1.9.0 is released!
+version: 1.9.0
+summary:
+thumbnail: +
+author: straight-shoota
+---
+
+We are delivering a new Crystal release with several bugfixes and improvements.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.9.0)
+and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for
+installation instructions.
+
+## Stats
+
+This release includes [XXX changes since 1.8.2](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.9.0)
+by XX contributors. We thank all the contributors for all the effort put into
```

Comment:
```suggestion
This release includes [166 changes since 1.8.2](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.9.0)
by 27 contributors. We thank all the contributors for all the effort put into
```